Makefile

```diff
@@ -489,6 +489,7 @@ ifeq ($(CI), true)
 endif
 
 _unit_test_coverage: $(REPORTS)
+	./hack/exclude_from_coverage.sh
```

Comment:
Unfortunately it seems impossible to do it in a clean way :sob:

Reply:
yikes. but if there's no other option I guess it is what it is